manticore/ethereum.py

```diff
@@ -287,21 +287,100 @@ def __mul__(self, reps):
     SUINT = SByte(32)
     SValue = None
 
+    @staticmethod
+    def _tuple_bytes_to_long(tpl):
+        return reduce(lambda acc, b: (acc << 8) + ord(b), tpl, 0L)
```

Comment:
Py3 requires `from functools import reduce` and `0L` -> `0`

Reply:
fixed in https://github.com/trailofbits/manticore/pull/875/commits/63efc4fb384bdd36f3e54d614b1c5e5625098f14